internal/provider/models/processors/test/route_test.go

```diff
@@ -102,6 +102,28 @@ func TestRouteProcessor(t *testing.T) {
 				ExpectError: regexp.MustCompile("(?s)Inappropriate value for attribute \"conditionals\": element 1: attribute.*\"label\" is required"),
 			},
 
+			// Error: label length too long
+			{
+				Config: GetCachedConfig(cacheKey) + `
+					resource "mezmo_route_processor" "my_processor" {
+						pipeline_id = mezmo_pipeline.test_parent.id
+
+						conditionals = [
+							{
+								expressions = [
+									{
+										field = ".status"
+										operator = "equal"
+										value_number = 200
+									}
+								]
+								label = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+							}
+						]
+					}`,
+				ExpectError: regexp.MustCompile("Invalid Attribute Value Length"),
```

Comment:
nitty, but we should check the property name too so that we know the invalid length is for the label.  Regex can span lines with the `(?s)` operator: https://github.com/mezmo/terraform-provider-mezmo/blob/17573000b9a495b4ded6c9b267dc54ffbc80da1a/internal/provider/models/alerts/test/threshold_test.go#L268